pulsar-client-reactive-jackson/src/test/java/org/apache/pulsar/reactive/client/jackson/PulsarReactiveClientModuleTests.java

```diff
@@ -187,6 +196,12 @@ void shouldSerDeserReactiveMessageConsumerSpec(Class<? extends ReactiveMessageCo
 				+ "  'acknowledgeAsynchronously' : true,\n"
 				+ "  'acknowledgeScheduler' : 'boundedElastic',\n"
 				+ "  'negativeAckRedeliveryDelay' : 30.000000000,\n"
+				+ "  'negativeAckRedeliveryBackoff' : {\n"
+				+ "    'className' : 'org.apache.pulsar.reactive.client.jackson.PulsarReactiveClientModuleTests$TestRedeliveryBackoff'\n"
```

Comment:
Any reason not to use `TestRedeliveryBackoff.class.getName()` here as well?

Reply:
No, fixed.